lib/OrdinaryDiffEqDifferentiation/src/derivative_wrappers.jl

```diff
@@ -309,8 +309,8 @@ function resize_jac_config!(cache, integrator)
             ad_left = autodiff_alg
         end
 
-        cache.jac_config = ([DI.prepare!_jacobian(
-                                   uf, cache.du1, config, ad, integrator.u)
+        SciMLBase.@reset cache.jac_config = ([DI.prepare!_jacobian(
```

Comment:
Is this an effective way to update `jac_config` in `cache`?

Reply:
No because matrices can't be resized.

Reply:
what does `SciMLBase.@reset` actually do?

Reply:
It's Acessors.jl